.github/workflows/master_update.yml

```diff
@@ -0,0 +1,18 @@
+name: master_update
+
+on:
+  push:
+    # branches:
+      # - master
+
+jobs:
+  release_image:
+    name: Build and publish to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: svtkachenko/schemathesis:latest
```

Comment:
hm, should it be `svtkachenko` or `kiwicom`?

Reply:
It is for a purpose of testing, see `[WIP]` in header :slightly_smiling_face: